components/src/utils/Cortex.js

```diff
@@ -37,6 +37,18 @@ export function cortexFetch(input, init) {
 
   return fetch(`${Config.cortexApi.path + input}`, requestInit)
     .then((res) => {
+      if (res.status === 401 || res.status === 403) {
+        localStorage.removeItem(`${Config.cortexApi.scope}_oAuthRole`);
+        localStorage.removeItem(`${Config.cortexApi.scope}_oAuthScope`);
+        localStorage.removeItem(`${Config.cortexApi.scope}_oAuthToken`);
+        localStorage.removeItem(`${Config.cortexApi.scope}_oAuthUserName`);
+        localStorage.removeItem(`${Config.cortexApi.scope}_b2bCart`);
+        localStorage.removeItem(`${Config.cortexApi.scope}_oAuthTokenAuthService`);
+        localStorage.removeItem(`${Config.cortexApi.scope}_keycloakSessionState`);
+        localStorage.removeItem(`${Config.cortexApi.scope}_keycloakCode`);
+        window.location.pathname = '/';
+        window.location.reload();
```

Comment:
I recall why we didn't add this immediately. This actually doesn't give the best experience. I'll need to revert it. 
This will also be triggered if an incorrect password is entered for login, and reloading on a 401 will cause the login modal to be dismissed upon reload.

Reply:
That makes sense, we don't want to affect user experience in other parts of the app.  We can add this to our deployments for now to address this issue, because that use case doesn't concern us too much.